ui/src/config/section/config.js

```diff
@@ -51,7 +51,7 @@ export default {
           label: 'label.remove.ldap',
           message: 'message.remove.ldap',
           dataView: true,
-          args: ['hostname'],
+          args: ['hostname', 'port', 'domainid'],
```

Comment:
@DaanHoogland Currently, remove LDAP option is provided in the details page, so can these details auto-picked from the LDAP details, instead prompting user to input?

LDAP details =>

<img width="1191" alt="LDAP_Details" src="https://user-images.githubusercontent.com/12028987/149872881-bf812499-e1a9-4303-a9f3-62fbac37ca90.png">

Old behavior =>

<img width="1187" alt="Removed_LDAP_Old" src="https://user-images.githubusercontent.com/12028987/149872912-273dcc83-8a85-464f-965e-3e15c096e9a6.png">

New behavior =>

<img width="1190" alt="Removed_LDAP_New" src="https://user-images.githubusercontent.com/12028987/149872924-ae05c786-384c-4825-81f2-3ba1994c16c4.png">